Makefile

```diff
@@ -59,7 +59,7 @@ cf-login-prod:
 	  cf-login
 
 DIRS        = api admin utils status
-TARGETS     = setup setup-dev build clean deploy deploy-dev test
+TARGETS     = setup setup-dev build check-vulnerabilities clean deploy deploy-dev test
```

Comment:
If we run `make check-vulnerabilities` from the project root, does this succeed? Per my comment below, I wonder if it will fail because the status project is missing an empty target. I could be wrong though.

Reply:
yeah it will fail on status.

we could extend this makefile target to include checking for out of date dependencies like I did when I was updating everything.


There's a ruby tool that I used called https://github.com/rubysec/bundler-audit

`pipenv check` and `npm audit` do the same for python and node projects, respectively.

Reply:
Not sure that i would want to fail CI on out of date deps though.

Reply:
Yeah I don't know about failing CI for that either, but good to know.

Reply:
Is this the prefered way to do things? I was planning to do it through this and have empty targets in `status/` but went the other direction and explicitly call `check-vulnerabilities` for each of the 3 directories instead; so using that method I can remove `check-vulnerabilities` from `TARGETS`